src/API/Google/AdsAssetGroupAsset.php

```diff
@@ -107,7 +108,7 @@ public function get_assets_by_asset_group_ids( array $asset_groups_ids, array $f
 			$asset_group_assets = [];
 			$asset_results      = ( new AdsAssetGroupAssetQuery() )
 				->set_client( $this->client, $this->options->get_ads_id() )
-				->add_columns( [ 'asset_group.id' ] )
+				->add_columns( [ 'asset_group.id', 'asset.youtube_video_asset.youtube_video_id' ] )
```

Comment:
Rather than adding this column to both places where the `AdsAssetGroupAssetQuery` is used, consider adding it as a default column, here: https://github.com/woocommerce/google-listings-and-ads/blob/dcb27c5c2e98970e5316aee8120d810640397eb9/src/API/Google/Query/AdsAssetGroupAssetQuery.php#L20

Reply:
Thank you @joemcgill , this looks like a proper place to add this new column. Made that change as per suggestion.